### DISCUSSION TITLE
How can I view the documentation after changing it?

### DISCUSSION THREAD

**Discussion by hecent** [python-pillow/Pillow#6490] [Q&A] [answered]
I've adjusted some docstrings but I worry it might break the rtf files, is there an easy way to view these?

**Comment by hugovk** [comment, marked as answer]
The CI builds a preview of each PR.

In the CI checks, look for:

"docs/readthedocs.org:pillow — Read the Docs build succeeded!"

And click Details on the right:

<img width="924" alt="image" src="https://user-images.githubusercontent.com/1324225/183988363-47f6cecb-783c-49b6-8dac-6f1ad4962e3b.png">

Which takes you to: https://pillow--6491.org.readthedocs.build/en/6491/

**Reply by radarhere** [reply]
For the record, other ways include
- signing up for https://readthedocs.org/, so that your Pillow fork has its own documentation subdomain, e.g. https://pillow-radarhere.readthedocs.io/en/latest/
- on your local machine, running `make doc`. You can then open `docs/_build/html/index.html`
- on your local machine, running `make docserve`, and then opening http://localhost:8000

**Reply by hecent** [reply]
Thanks
